docs/Passing-Devices-with-Clear-Containers.md

```diff
@@ -0,0 +1,65 @@
+# Using devices with Clear Containers
+
+Host devices can be passed to a container with Docker using `--device` option
+with docker run. An example of passing a device with docker is given below:
+```
+$ sudo docker run -it --device=/dev/sdb debian
+```
+Clear Containers has support for passing the following devices to the
+container with `--device`.
+
+## Virtual Function I/O (VFIO) Devices (PCI Passthrough)
+
+The VFIO driver is a framework for exposing direct device access to userspace, 
+in a secure, Input–Output Memory Management Unit (IOMMU) protected environment.
+A device can be directly passed to a Clear Containers container using VFIO. 
+
+For information on VFIO, please refer to:
+ - https://www.kernel.org/doc/Documentation/vfio.txt
+ - https://www.linux-kvm.org/images/b/b4/2012-forum-VFIO.pdf
+
+Devices can pe passed to a Clear Container container using VFIO
+as shown below:
+
+```
+$ sudo docker run -it --device=/dev/vfio/16 centos/tools bash
+```
+The user is expected to unbind the device from its host driver, bind the device
+to `vfio-pci` driver and then pass the IOMMU group that the device belongs to on
+the docker command line.
+For detailed steps on how to do this, please refer to
```

Comment:
For detailed steps on using VFIO to pass devices to a Clear Container container, see...

Reply:
Its not just passing devices using VFIO, I was referring to binding and unbinding devices.

Reply:
Understood - If this does not change meaning, suggested rewrite: For detailed steps to bind and unbind devices, see...